mvpa2/support/afni/lib_prep_afni_surf.py

```diff
@@ -286,19 +311,75 @@ def yesno2bool(d, k): # dict, key
             c['outvol_ext'] = '+' + c['outvol_view']
             c['outvol_fullext'] = c['outvol_ext'] + '.HEAD'
 
-
     return c
 
+
+
+def _ensure_expvol_is_plump(expvol):
+    '''Helper function that raises an error if the experimental volume
+        is oblique, because such volumes are not supported (yet)'''
+    cmd = '3dinfo "%s"' % expvol
+    info = utils.cmd_capture_output(cmd)
```

Comment:
@yarikoptic I agree that having tests for ``ensure_expvol_is_plump`` would be useful. However currently coverage for all functionality in ``lib_prep_afni_surf`` is close to (if not exactly) non-existent. Some integration tests would probably be a better approach compared to writing quite a bit of test code (including tests for checking presence of afni, 3dinfo, ...) with a relative small increase in test coverage. Maybe we can open an issue about coverage for this library and in the meantime just merge?